cdap-data-fabric/src/main/java/co/cask/cdap/data2/util/hbase/HBaseTableUtil.java

```diff
@@ -533,6 +533,13 @@ public String toHBaseNamespace(Id.Namespace namespace) {
   public abstract Map<String, TableStats> getTableStats(HBaseAdmin admin) throws IOException;
 
   /**
+   * Gets the system configuration table prefix
```

Comment:
can you add more detail here?  What is the system configuration table prefix and what's an example?